content/guide/dsl-import-resolver.md

```diff
@@ -82,8 +82,8 @@ In case this url cannot be resolved, it will try to apply the third rule by repl
 If this url, also, cannot be resolved, it will try to resolve the original url, i.e. `http://prefix2.suffix2.org`
 In case that all the resolve attempts will fail, a `DSLParsingLogicException` will be raise.
 
-## Configuring custon rules for the default resolver:
```

Comment:
line 50: "the rules" -> "rules", because the reader doesn't know what rules you're reffering to